.github/workflows/test.yml

```diff
@@ -292,7 +292,7 @@ jobs:
         run: sudo lxc exec test-container -- sudo bash -c "cat /tmp/authorized_keys.user >> /root/.ssh/authorized_keys"
 
       - name: Install authorized_keys for ubuntu user in container
-        run: sudo lxc exec test-container -- cp -ai /root/.ssh/authorized_keys /home/ubuntu/.ssh
+        run: sudo lxc exec test-container -- cp -af /root/.ssh/authorized_keys /home/ubuntu/.ssh
```

Comment:
This is an empty file, so `cp -ai` tried to prompt.

Reply:
The real question is: why is it suddenly empty.

Removing something because it is failing now without explanation is not helpful.

Reply:
It just is:

```bash
$ sudo lxc launch ubuntu:22.04 test-container -c security.nesting=true
Creating test-container
Starting test-container                   
$ sudo lxc exec test-container -- ls -ld /home/ubuntu/.ssh/authorized_keys
-rw------- 1 ubuntu ubuntu 0 Mar  2 11:10 /home/ubuntu/.ssh/authorized_keys
```

Reply:
The file is created [here](https://github.com/andreasscherbaum/ansible-lxc-ssh/blob/104f06491d34cd2479ad455f1d7450ee576526e2/.github/workflows/test.yml#L284), and is not supposed to be empty. Something is wrong.

Reply:
`root/.ssh/authorized_keys` is not empty. But `/home/ubuntu/.ssh/authorized_keys` exists and is empty even in a new container, see above. This causes `cp -ai` to fail.

Reply:
Oh, gotcha. So the file exists now (and did not exist in earlier versions). Using `cp -ai` caught this problem just fine, but now you are removing the copy, instead of fixing the problem.

Reply:
I'm not removing the copy, I'm replacing `cp -ai` with `cp -af`. Correct about the rest.